### ISSUE TITLE
Filter entity lists by name

### ISSUE THREAD

**Issue by latibro** [latibro/Automation-Mod#1]
Possibility of filtering the EntityListAPI.

As a start the filteren could be on "type"

**Comment by latibro** [comment]
Maybe something like this

```lua
local rollingstocks = entities.filterByType("immersiverailroading:rollingstock")
```

**Comment by latibro** [comment]
Maybe something like this

```lua
local rollingstocks = entities.filterByType("immersiverailroading:rollingstock")
```

**Comment by latibro** [comment]
In 0.1.0-alpha.4 filter by name has been added

```lua
entities.filterByName("Cow")
```

Name is is:

*   For players : Username
*   Entities with name (from nametag)
*   Everything else is kinda a generic name from MC (Cow, Pig, Chicken etc) or what other mods has as generic names. 

This is the same name as shown in game

**Comment by latibro** [comment]
Has changed this to filter on name, and closing this issues as that is included in 0.1.0.alpha.4

We may add a new issue if we still would need to filter on type of some sort.
